de/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="de">
+<head>
+  <title>Frag nicht zum Fragen, frag einfach</title>
+  <meta property="og:title" content="Frag nicht zum Fragen, frag einfach" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
+  <meta property="og:locale" content="de_DE" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Frag nicht zum Fragen, frag einfach</h1>

```

Comment:
```suggestion
    <h1>Frag nicht, ob du Fragen darfst, frag einfach</h1>
```

Not that my translation is any better, this is really hard to translate well.